Crashlytics/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+
+- [fixed] Force validation or rotation of FIDs for FirebaseSessions.
```

Comment:
Move into the 10.22.0 section

Reply:
Thanks for the review!Is this one time thing because I didn't catch up the log change or in the future we always add the release notes to upcoming release version?

Reply:
We haven't had a consistent practice - maybe we should.

When I added my release note today, I updated the Unreleased to 10.22.0 in advance of tomorrow's code freeze.